src/Core/ViewHelper/TagBuilder.php

```diff
@@ -47,7 +47,7 @@ class TagBuilder
      * Constructor
      *
      * @param string $tagName name of the tag to be rendered
```

Comment:
Can you move the Fluid handling of the TagBuilder to a separate PR please?

Reply:
Not sure what you precisely mean with this - could you elaborate? Do you want the capability to render closures and the chaining additions in a separate PR or something else?